package.json

```diff
@@ -43,7 +43,7 @@
     "esbuild": "^0.25.0",
     "esbuild-jest": "^0.5.0",
     "eslint": "^9.2.0",
-    "eslint-plugin-react-hooks": "5.2.0-canary-662957cc-20250221",
```

Comment:
why are we on a canary? :thinking:

Reply:
From https://github.com/keystonejs/keystone/pull/9497:
> Also, the reason the version of eslint-plugin-react-hooks is 5.2.0-canary-662957cc-20250221 is because the proper ESLint 9 support isn't in the latest normal version of eslint-plugin-react-hooks